### PR TITLE
Migrate bower to yarn

### PR DESCRIPTION
**Pull request by nicolasleger**
Issue https://github.com/openfoodfacts/openfoodfacts-server/issues/922

See build at https://travis-ci.org/nicolasleger/openfoodfacts-server

**Now use `yarn install` (need to be installed, version >= 1) instead of `npm install`.**

### CODE REVIEW

**Comment by nicolasleger** [comment]
@stephanegigandet @teolemon Hi, can you enable Travis build on Pull Requests to ensure build and tests?

File: .travis.yml [hunk 1]
```diff
@@ -21,10 +23,11 @@ addons:
 sudo: false
 before_install:
   - cpanm --local-lib=~/perl5 local::lib && eval $(perl -I ~/perl5/lib/perl5/ -Mlocal::lib)
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH="$HOME/.yarn/bin:$PATH"
 install:
   - cpanm --quiet --installdeps --notest --skip-satisfied .
-  - npm install
```
Comment (hangy) [inline .travis.yml]: Shouldn't `npm install` stay because of the jshint dependency?

**Comment by hangy** [comment]
I still have #708 and #725 open that would probably cause conflicts with this. Maybe those could be reviewed first?

**Comment by hangy** [comment]
@nicolasleger I re-enabled PR builds. One problem is that the "new crowdin translations" PR also causes builds, I think. 🙍‍♂️

File: .travis.yml (hunk 1 above)
Comment (nicolasleger) [reply .travis.yml]: `yarn` is auto-detected by Travis and install `jshint` by reading `package.json`
See the end of the success build logs at https://travis-ci.org/nicolasleger/openfoodfacts-server

**Comment by nicolasleger** [comment]
Good @hangy !
Ask again the requested reviewers.

However it can be done in parallel of this one since it's compatible if you add `gulp` and adapt `scripts` commands.
But this one is not mandatory and urgent yet, it is still a warning.

**Comment by hangy** [comment]
Merci @nicolasleger 😄

**Comment by nicolasleger** [comment]
Thanks for your review and work.

**Pull request merged**